src/main/java/org/folio/rest/impl/ReceivingHistoryAPI.java

```diff
@@ -21,11 +23,13 @@
 import io.vertx.core.Handler;
 
 public class ReceivingHistoryAPI implements OrdersStorageReceivingHistory {
+  private static final Logger log = LogManager.getLogger();
 
   @Override
   @Validate
   public void getOrdersStorageReceivingHistory(String query, int offset, int limit, String lang, Map<String, String> okapiHeaders,
       Handler<AsyncResult<Response>> asyncResultHandler, Context vertxContext) {
+    log.info("getOrdersStorageReceivingHistory, query={query}");
```

Comment:
It seems that query supposed to be as a paramtert in this log?

Reply:
Indeed, thanks for catching that !